### DISCUSSION TITLE
Schedule

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#48] [Team Posts]
Dear Participants,

(This message is doubling an earlier announcement to the Participants mailing list, which ended up in spam for some of us. Someone please leave a comment here confirming you got this?.. Thx.)

We have now settled on the final program for this year's summit, and provided the usual top-level page for the summit, at
https://github.com/delph-in/docs/wiki/Virtual2021Top
Please look over the schedule if you are presenting to confirm that we have things set up correctly.

We will send out information in a few days about the Zoom link(s) to use during the week.  We're looking forward to meeting up soon.

Your program committee,
Olga and Dan

**Comment by olzama** [comment]
Dear @delph-in/participants -- how about this, did you get a notification now? :)

**Comment by guyemerson** [comment]
I was notified both times by email.  Each person can choose their own notification settings.

**Comment by olzama** [comment]
...but hopefully the mention will result in everyone in the team getting notified?

**Comment by guyemerson** [comment]
Notifications can be switched off, but I believe they are enabled by default.

**Comment by YanfeiFang** [comment]
So sorry for replying late. I’ve received the message.

Fang Yanfei

**Comment by YanfeiFang** [comment]
Sorry for replying late. I’ve got the message. Thank you for your kind remind and consideration!

**Comment by leoalenc** [comment]
> So sorry for replying late. I’ve received the message.
> 
> Fang Yanfei
> …

@olzama, thanks, I've received all notifications, sorry for the late response.

**Comment by olzama** [comment]
Thanks, everyone!! I just needed a couple confirmations, to make sure the info is reaching people generally :). Looking forward to seeing everyone soon!!

**Comment by EricZinda** [comment]
Got it!
